### PR TITLE
Story CORE-583 | [BETA] Configure routes writing on LB-sidecar

### PR DESCRIPTION
**Pull request by LucasStarlingdePaulaSalles**
# Description

Kind: Story

Section: Pkg Load Balancer, Environment utils

Summary: implemented route request handler on lb sidecar for route communication feature

## Changelog
- feature:
  - lbsidecar routes request writer handler
  - route redirect resolution
  - route address port resoution

### CODE REVIEW

File: pkg/sidecars/lbsidecar/handlers_test.go [hunk 1]
```diff
@@ -292,4 +292,122 @@ func deleteMockEnvVars() {
...
+		func(w http.ResponseWriter, r *http.Request) {
+
+			var receivedData string
+			err := json.NewDecoder(r.Body).Decode(&receivedData)
+			if err != nil {
+				rest.ERROR(w, fmt.Errorf("error while reading msg body"))
+				return
+			}
+
+			// fmt.Printf("receivedData = %v\n", string(body))
```
Comment (luiz-couto) [inline pkg/sidecars/lbsidecar/handlers_test.go]: I think you can remove this comment

File: pkg/environment/utils.go [hunk 2]
```diff
@@ -75,6 +76,19 @@ func GetResolvedChannel(channel string, inputChan, outputChan []brokers.ChannelB
 	).InvalidChannel()
 }
 
+// GetRouteData returns the data of a resolved route
+func GetRouteData(route string) (*meta.RouteConnection, error) {
+	value, exists := os.LookupEnv(route + "_ROUTE")
+	if !exists {
+		return nil, ierrors.New("invalid route: %s", route).BadRequest()
```
Comment (luiz-couto) [inline pkg/environment/utils.go]: I think it would be better to change the error message to something like: "route %v not found in enviroment variables"

**Review by luiz-couto — APPROVED** [review]

**Review by luiz-couto — COMMENTED** [review]
Some small changes. You also need to verify the test that is not passing

**Review by luiz-couto — CHANGES_REQUESTED** [review]
Some changes

**Review by luiz-couto — APPROVED** [review]

**Review by lucasABLima — APPROVED** [review]

**Pull request merged**